bridgetown-core/test/test_filters.rb

```diff
@@ -31,6 +31,23 @@ class SelectDummy
     def select; end
   end
 
+  M = Struct.new(:message) do
```

Comment:
I actually prefer this struct style personally 👏

Reply:
+1! The inheriting of an initialized strict feels...weird, and I'm all for keeping Ruby weird...